app/src/main/java/com/example/coinage/models/Spending.java

```diff
@@ -0,0 +1,36 @@
+package com.example.coinage.models;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+import com.parse.ParseUser;
+
+@ParseClassName("Spending")
+public class Spending extends ParseObject {
```

Comment:
Does a "Spending" represent a user's spending limit for a given category? If so, is there a more informative name we could give it?